src/pages/FaqPage/__tests__/hashScroll.test.tsx

```
@@ -0,0 +1,94 @@
+// @vitest-environment jsdom
+import React from "react";
+import { describe, it, expect, vi, afterEach } from "vitest";
+import { render } from "@testing-library/react";
+
+const useLocationMock = vi.fn();
+var scrollToMock: any;
+
+vi.mock("react-router", () => ({
+  Link: ({ children }: any) => <a>{children}</a>,
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)